modules/System Admin/systemSettingsProcess.php

```diff
@@ -32,409 +32,75 @@
     header("Location: {$URL}");
     exit;
 } else {
-    //Proceed!
-    $absoluteURL = $_POST['absoluteURL'];
-    $absolutePath = $_POST['absolutePath'];
-    $systemName = $_POST['systemName'];
-    $indexText = $_POST['indexText'];
-    $organisationName = $_POST['organisationName'];
-    $organisationNameShort = $_POST['organisationNameShort'];
-    $organisationEmail = $_POST['organisationEmail'];
-    $organisationLogo = $_POST['organisationLogo'];
-    $organisationBackground = $_POST['organisationBackground'];
-    $organisationAdministrator = $_POST['organisationAdministrator'];
-    $organisationDBA = $_POST['organisationDBA'];
-    $organisationHR = $_POST['organisationHR'];
-    $organisationAdmissions = $_POST['organisationAdmissions'];
-    $pagination = $_POST['pagination'];
-    $timezone = $_POST['timezone'];
-    $country = $_POST['country'];
-    $firstDayOfTheWeek = $_POST['firstDayOfTheWeek'];
-    $analytics = $_POST['analytics'];
-    $emailLink = $_POST['emailLink'];
-    $webLink = $_POST['webLink'];
-    $defaultAssessmentScale = $_POST['defaultAssessmentScale'];
-    $installType = $_POST['installType'];
-    $statsCollection = $_POST['statsCollection'];
-    $passwordPolicyMinLength = $_POST['passwordPolicyMinLength'];
-    $passwordPolicyAlpha = $_POST['passwordPolicyAlpha'];
-    $passwordPolicyNumeric = $_POST['passwordPolicyNumeric'];
-    $passwordPolicyNonAlphaNumeric = $_POST['passwordPolicyNonAlphaNumeric'];
-    $sessionDuration = $_POST['sessionDuration'];
-    $currency = $_POST['currency'];
-    $backgroundProcessing = $_POST['backgroundProcessing'] ?? 'Y';
-
-    //Validate Inputs
-    if ($absoluteURL == '' or $systemName == '' or $organisationLogo == '' or $indexText == '' or $organisationName == '' or $organisationNameShort == '' or $organisationEmail == '' or $organisationAdministrator == '' or $organisationDBA == '' or $organisationHR == '' or $organisationAdmissions == '' or $pagination == '' or (!(is_numeric($pagination))) or $timezone == '' or $installType == '' or $statsCollection == '' or $passwordPolicyMinLength == '' or $passwordPolicyAlpha == '' or $passwordPolicyNumeric == '' or $passwordPolicyNonAlphaNumeric == '' or $firstDayOfTheWeek == '' or ($firstDayOfTheWeek != 'Monday' and $firstDayOfTheWeek != 'Sunday' and $firstDayOfTheWeek != 'Saturday') or $currency == '') {
```

Comment:
I notice that there is no validation. I guess if a field is missing is just does not get updated, but then it does not return any information to the user. Just curious...

Reply:
Good question. Will push a commit to address required fields 👍